src/main/java/com/spotify/folsom/client/DefaultRawMemcacheClient.java

```diff
@@ -74,6 +74,8 @@
   private static final EventLoopGroup EVENT_LOOP_GROUP =
       new NioEventLoopGroup(0, DAEMON_THREAD_FACTORY);
 
+  static final AtomicInteger GLOBAL_CONNECTION_COUNT = new AtomicInteger();
```

Comment:
I'd wrap this in a static getter returning an int rather than exposing the raw AtomicInteger (even with default access)

Reply:
That's sane!